.gitignore

```diff
@@ -15,4 +15,8 @@
 # Directories
 .*
 !.gitignore
-/github-connector/vendor
\ No newline at end of file
+/github-connector/vendor
+/.vscode
+
+# Gopkg
+Gopkg.*
```

Comment:
Why you're ignoring this?

Reply:
I copied it from master, but after some discussion with team it turns out that it needs to be deleted - I will change it in a second